### PR TITLE
minor rewording to current state, small higher math example

### PR DESCRIPTION
**Pull request by dginev**
Making my way towards the examples, part 2. Summary:

1. Took a moderate clarifying posture to the use of "semantic", tried to straighten up the wording here and there.

2. I removed:
   > Many of the techniques used to make text accessible can be applied to math, but many don't work well
   - since I think it would need significant expansion (which techniques work? +example; which don't? +example) to keep in.

3. Also removed the mention of "bevelled", since the spec says it's depracted, and mentioned the Kronecker symbol as a neat non-fraction example for positive line-thickness.

4. Added a higher math example for `L2` which is a rather well-known non-power use of the supercript-2.

We can discuss/debate other of the edits, I may have redirected some of the subtle implications, mostly intentionally.

### CODE REVIEW

File: gap-analysis/index.md [hunk 1]
```diff
@@ -49,22 +49,23 @@ The platform APIs in general have not provided much support for mathematical not
...
+Higher mathematics adds an additional tier of complexty. In it, even $x^2$ can be ambiguous, and may warrant a different reading than ordinary. One such case is $L^2$ read simply "L2" in the domain of Lebesgue spaces. In such texts, the author can often find themselves to be one of a only a handful of practitioners who have full grasp of their technical terminology, making manual annotation a frequent need for good AT.
+
+Some notations, such as fractions and square roots, may require bracketing words or tones to indicate the start and end of an expression, for readers who can't see the presentation. For example, $\sqrt{x+1}$ is unambiguously spoken as "the square root of x plus 1 end root". Without these bracketing words, "the square root of x plus 1" could also be interpreted as $\sqrt{x}+1$. However, for someone who is dyslexic and uses AT, the extra words are a distraction and shouldn't be used. This difference between the needs of users means that a real solution should allow flexibility based on the reader -- literal text strings annotated by authors may be inappropriate for some readers.
```
Comment (NSoiffer) [inline gap-analysis/index.md:68]: > ...means that a real solution...

"real" sounds weird -- as opposed to a "fake" solution. I suggest "good" instead

Comment (NSoiffer) [reply gap-analysis/index.md:68]: Maybe "full" (solution)? "production-grade"  doesn't seem appropriate. It is true for research projects, prototypes, etc.

Comment (dginev) [reply gap-analysis/index.md]: "full" is a great fit actually, done!

File: gap-analysis/index.md [hunk 2]
```diff
@@ -49,22 +49,23 @@ The platform APIs in general have not provided much support for mathematical not
...
 
-Some notations such as fractions and square roots sometimes require bracketing words or tones to indicate the start and end of some notations for someone who can't see the notation. For example, $\sqrt{x+1}$ is unambiguously spoken as "the square root of x plus 1 end root". Without these bracketing words, "the square root of x plus 1" could also be interpreted as $\sqrt{x}+1$. However, for someone who is dyslexic and uses AT, the extra words are a distraction and shouldn't be used. This difference between the needs of users means that any solution that is adopted should allow flexibility based on the reader -- simple text strings generated by authors may be in appropriate for some readers. 
+Higher mathematics adds an additional tier of complexty. In it, even $x^2$ can be ambiguous, and may warrant a different reading than ordinary. One such case is $L^2$ read simply "L2" in the domain of Lebesgue spaces. In such texts, the author can often find themselves to be one of a only a handful of practitioners who have full grasp of their technical terminology, making manual annotation a frequent need for good AT.
```
[the remark this answers is not in the document]
Comment (dginev) [reply gap-analysis/index.md]: still getting used to that acronym, thanks. Done!

**Pull request merged**